configuration/pipeline_config.json

```diff
@@ -0,0 +1,71 @@
+{
+  "PipelineName": "EU-PCVE",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/csap-text-it-token.txt",
+      "ActivationFlowNames": [
+        "csap_eu_pcve_s01e01_activation"
+      ],
+      "SurveyFlowNames": [
+        "csap_eu_pcve_demog"
```

Comment:
@as2388 cc @DanielMwendwa  do we have g-cloud demog archives for previous projects that we should add here or should we use the textit versions.

Reply:
Yes, we basically need to take the configuration from USAID-IBTCI and replace the activation flows there with the ones here (so the configuration file ends up being very long)

Reply:
@as2388 kindly work on that, thanks.

Reply:
thanks, updated.